cmflib/commands/init/amazonS3.py

```diff
@@ -0,0 +1,125 @@
+###
+# Copyright (2022) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+
+#!/usr/bin/env python3
+import argparse
+import os
+import subprocess
+import shlex
+
+from cmflib.cli.command import CmdBase
+from cmflib.cli.utils import create_cmf_config
+
+
+class CmdInitAmazonS3(CmdBase):
+    def run(self):
+        cmf_config = "./.cmfconfig"
+        if self.args.cmf_server_ip:
+            create_cmf_config(cmf_config, self.args.cmf_server_ip)
+        else:
+            if not os.path.exists(cmf_config):
+                create_cmf_config(cmf_config, "http://127.0.0.1:80")
+        abs_path = None
+        file = "git_initialize.sh"
+        # find absolute path for git_initialize.sh
+        for root, dirs, files in os.walk(os.path.dirname(__file__)):
+            for name in files:
+                if name == file:
+                    abs_path = os.path.abspath(os.path.join(root, name))
+        # executing git_initialize.sh
+        result = subprocess.run(
+            ["sh", f"{abs_path}", f"{self.args.git_remote_url}"],
+            stdout=subprocess.PIPE,
+            stderr=subprocess.STDOUT,
+            text=True,
+        )
+        if result.stdout:
+            print(result.stdout)
+        file = "dvc_script_amazonS3.sh"
+        # find absolute path for dvc_script_amazonS3.sh
+        for root, dirs, files in os.walk(os.path.dirname(__file__)):
+            for name in files:
+                if name == file:
+                    abs_path = os.path.abspath(os.path.join(root, name))
+        # executing dvc_script_amazonS3.sh
+        result = subprocess.run(
```

Comment:
same here , check abs_path not none before proceeding

Reply:
Done.